src/screens/Followings/elements/RenderList/index.js

```diff
@@ -3,39 +3,39 @@ import React from 'react';
 import {Image, StyleSheet, Text, View} from 'react-native';
 import GlobalButton from '../../../../components/Button/GlobalButton';
 
-import {colors} from '../../../../utils/colors';
 import {fonts} from '../../../../utils/fonts';
+import {COLORS} from '../../../../utils/theme';
 
 // data needed name, description, image
 const styles = StyleSheet.create({
   buttonFollow: {
     width: 88,
     height: 36,
     flexDirection: 'row',
-    backgroundColor: colors.bondi_blue,
+    backgroundColor: COLORS.holyTosca,
     borderRadius: 8,
     alignItems: 'center',
     justifyContent: 'center'
   },
   textButtonFollowing: {
     fontSize: 12,
-    color: colors.bondi_blue,
+    color: COLORS.holyTosca,
     fontFamily: fonts.inter[600],
     fontWeight: 'bold'
   },
   textButtonFollow: {
     fontSize: 12,
-    color: colors.white,
+    color: COLORS.white,
     fontFamily: fonts.inter[600],
     fontWeight: 'bold'
   },
   profilepicture: {
     width: 48,
     height: 48,
-    borderColor: colors.lightgrey,
+    borderColor: COLORS.lightgrey,
     borderRadius: 24,
     borderWidth: 1,
-    // backgroundColor: colors.bondi_blue,
+    // backgroundColor: COLORS.holyTosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `buttonFollow` and `textButtonFollowing` styles have been changed from `colors.bondi_blue` to `COLORS.holyTosca`. Ensure that this color change aligns with the overall theme and design guidelines of your application. If it does, then this change is fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->